wallet.go

```diff
@@ -1122,6 +1124,102 @@ func (w *Wallet) DumpWIFPrivateKey(addr btcutil.Address) (string, error) {
 	return wif.String(), nil
 }
 
+// ImportAddress imports a watch-only address to the wallet.
+func (w *Wallet) ImportAddress(addr btcutil.Address, bs *waddrmgr.BlockStamp,
+	rescan bool) (string, error) {
+
+	// The starting block for the key is the genesis block unless otherwise
```

Comment:
This needs a check/error if it's not for the current network the manager is using.

Something like:

```Go
if !addr.IsForNet(activeNet.Params) {
	// error...
}
```